lib/src/modules/navigationItems/NavigationItem.js

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
+import cx from 'classnames';
+import { NavigationContext } from './NavigationProvider';
+
+export function NavigationItem({ children, id, className }) {
+  const { setActiveTabId, activeTabId } = useContext(NavigationContext);
+  const isActive = activeTabId === id;
+
+  const classNames = cx(className, 'navigation--item', {
+    active: isActive
+  });
+  return (
+    <li className={classNames} role="none">
+      <button
```

Comment:
🚩  is it a little strange how this is called `NavigationItem` but anchor tags aren't used?